src/server/models/userModel.ts

```diff
@@ -13,21 +13,24 @@ export const TOKEN_COLLECTION_NAME = 'user-tokens';
 
 export const REQUIRED_SCOPES = ['repo'];
 
-export interface LoginDetails {
+export interface UserRecord {
+  githubToken: string;
+  scopes: string[];
   username: string;
-  avatarUrl: string|null;
   fullname: string|null;
-  githubToken: string;
-  scopes: string[]|null;
+  avatarUrl: string|null;
   lastKnownUpdate: string|null;
+  lastViewed?: number;
```

Comment:
This variable name is confusing - sounds like the last time the user viewed the dashboard. Instead this is the first time the dot feature was enabled. Maybe `dotsEnabledAt`, `lastViewedFeatureEnabledAt` or `feature-lastViewed-enabledAt`?